presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PackageController.java

```diff
@@ -203,17 +201,19 @@ public Response stop(StopType stopType)
     private void gracefulStop()
             throws PrestoManagerException
     {
+        URI uri;
         try {
-            UriBuilder uriBuilder = fromUri(format("http://localhost:%s", informer.getPrestoPort())).path("/v1/info/state");
-            Response response = client.target(uriBuilder.build()).request(TEXT_PLAIN)
-                    .buildPut(entity(GSON.toJson("SHUTTING_DOWN"), APPLICATION_JSON)).invoke();
-            if (response.getStatus() != 200) {
-                throw new PrestoManagerException("Failed to stop presto gracefully");
-            }
+            uri = fromUri("http://localhost").port(informer.getPrestoPort())
+                    .path("/v1/info/state").build();
         }
         catch (IOException e) {
             throw new PrestoManagerException("Failed to get Presto port number.", e);
         }
+        Response response = client.target(uri).request(TEXT_PLAIN)
+                .buildPut(entity("\"SHUTTING_DOWN\"", APPLICATION_JSON)).invoke();
```

Comment:
This request to Presto should contain the body in `JSON` format.
For some reason Jersey client is not automatically serializing string to JSON format.
I should either have this quote marks or use `ObjectMapper` to serialize the string `SHUTTING_DOWN` to json format.